### DISCUSSION TITLE
'End' ⇄ 'Start' of segment at its creation stage

### DISCUSSION THREAD

**Discussion by evd316** [ReVanced/revanced-discussions#1061] [Question]
Good day!

I am one of the UKR translators/proofreaders in some projects on GitHub & Crowdin. Pleease swap the function of the 'end' and 'start' of segment at its creation stage (look screeshot), as it is non-logical and VERY uncomfortable:
![зображення_viber_2024-08-14_02-40-37-859](https://github.com/user-attachments/assets/87d66147-f713-403b-9104-11f507975808)

**Comment by LisoUseInAIKyrios** [comment]


**Reply by LisoUseInAIKyrios** [reply]
What you are seeing is a layout issue.

The vertical button layout is not showing for me and the dialog buttons are correctly shown as horizontal.

It seems the buttons can be shown vertical if the button text is too large (or maybe the device font is set too large and the screen size is too small).  I see some ways to try and always force horizontal buttons, but would first need a way to reproduce the issue you are seeing.

![yt](https://github.com/user-attachments/assets/1a5a7ca7-a633-4f6b-8ed3-17e3576d1630)

**Reply by LisoUseInAIKyrios** [reply]
It appears that forcing a horizontal button layout would require a custom dialog layout.  Not worth the effort.

Or you can change the device font size to a slightly smaller value and that should fix it without any code changes.

**Comment by evd316** [comment]
I use mod 17.34.36 for Android 7.0, is this not fixable? Version not supported?
Default font size, 6.0" screen 1280×720.

**Reply by oSumAtrIX** [reply]
Unofficial/outdated patches, so no, use the latest patches

**Reply by evd316** [reply]
"use the latest patches" on my mod 17.34.36 for Android 7.0? I have 2.160.10 – the latest in the world for my configuration.
Or do you mean buy a new phone and "use the latest patches"?

**Reply by LisoUseInAIKyrios** [reply]
You are using some unofficial patches the ReVanced project has no control over (version 17.x is not supported).

Buying a new phone and using official ReVanced won't fix this UI issue either (unless maybe you buy a new phone with a huge screen). 

You can change your device settings to use a smaller system font and that might fix the issue for you.

Just ignore the issue.

**Comment by kitadai31** [comment]
Yes, it's simply your screen size issue.

![Screenshot_20240814-164807](https://github.com/user-attachments/assets/df51b794-1c84-4436-bb18-71749dfcf6ea)
![Screenshot_20240814-164820](https://github.com/user-attachments/assets/f9b31cf6-d014-4c1b-ba5d-74fba3c3753b)
